src/pyscaffold/shell.py

```diff
@@ -132,10 +149,10 @@ def get_git_cmd(**args):
     Args:
         **args: additional keyword arguments to :obj:`~.ShellCommand`
     """
-    if sys.platform == "win32":  # pragma: no cover
+    if IS_WINDOWS:  # pragma: no cover
         # ^  CI setup does not aggregate Windows coverage
-        for cmd in ["git.cmd", "git.exe"]:
```

Comment:
I understand from this removal that `"git.cmd"` is no longer necessary, and that we expect any distribution of git for Windows to add a `.exe` to the PATH, right?

Reply:
Yes. I was actually trying to find `git.cmd` by googling but had no luck. There is a "git CMD" but no "git.cmd" so maybe it was a mistake? Also setuptools-scm doesn't seem to be doing this, so at least we would fail both.